main.go

```diff
@@ -182,92 +178,128 @@ func mainCore() error {
 		return fmt.Errorf("Unable to get block from node: %v", err)
 	}
 
+	var fetchToHeight = int64(math.MaxInt32)
 	if usePG {
```

Comment:
Comment that this is the default in lite mode meaning always RPC directly for blocks